cmake/CPackage.cmake

```diff
@@ -17,11 +18,15 @@ macro(package is_one)
         set(CPACK_COMPONENTS_GROUPING ONE_PER_GROUP)
     endif()
 
+    set(CPACK_DEBIAN_PACKAGE_MAINTAINER "vesoft")
+    set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "A distributed, scalable, lightning-fast graph database")
     set(CPACK_DEB_COMPONENT_INSTALL YES)
     set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
     set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://github.com/vesoft-inc/nebula/releases")
     set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA ${CMAKE_CURRENT_SOURCE_DIR}/package/postinst)
 
+    set(CPACK_RPM_PACKAGE_GROUP "vesoft")
```

Comment:
`VEsoft Inc.`

Reply:
agreed

Reply:
The head of the file is `vesoft inc`. I think it's unified it as `vesoft inc`.